ChainReactionBindings/TestBase/Pages/WishlistPopupPage.cs

```diff
@@ -0,0 +1,66 @@
+﻿using OpenQA.Selenium;
+using OpenQA.Selenium.Support.UI;
+
+namespace ChainReactionBindings.TestBase.Pages
+{
+	class WishlistPopupPage : BasePage
+	{
+		public string AddPopupId = ".addto_wishlist_pop.addcart";

```

Comment:
Please make all the ids private, we should not give access to them outside the PageObject